include/seqan3/alphabet/concept.hpp

```diff
@@ -78,7 +78,7 @@ void to_rank(args_t ...) = delete;
 //!\brief Functor definition for seqan3::to_rank.
 struct to_rank_fn
 {
-private:
+public:
```

Comment:
why do the impl function need to be public? As I understand it, they are only used by the `operator()`?

Reply:
Because the require statement in gcc only allows public statements; clang on the other hand allows this code.

Not sure if this is a bug/workaround, but @h-2 said that I should not bother, because it is `detail` anyway.